src/wcomms/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+add_library(wcomms SHARED)
```

Comment:
Why do you need this module? You may have these classes in the existing module `worker`